features/support/hooks.js

```diff
@@ -124,6 +124,32 @@ Before({ tags: "@createBugStory" }, async function(){
     this.createBugStoryHook = true;
 });
 
+Before({ tags: "@createFeatureStory" }, async function(){
```

Comment:
Se parece mucho al de @createBugStory .. quiza haya que refactorizar

Reply:
El hook de crear un bug lo puedes editar así para que también pueda crear feature story `Before({ tags: "@createBugStory or @createFeatureStory" }, async function(scenario){
    if (!this.createBugStoryHook){
        const addStoryButton = await DriverFactory.myDriver.wait(until.elementLocated(StoriesTab.addStoryButton));
        await DriverFactory.myDriver.wait(until.elementIsVisible(addStoryButton), configuration.browser.timeout);
        await addStoryButton.click();
        const storyTitleTextField = await DriverFactory.myDriver.wait(until.elementLocated(StoryPanel.storyTitleTextField));
        const storyTypeDropdown = await DriverFactory.myDriver.wait(until.elementLocated(StoryPanel.storyTypeDropdown));
        this.firstStoryName = RandomValues.alphanumeric(6);       
        await storyTitleTextField.sendKeys(this.firstStoryName);
        await storyTypeDropdown.click();
        const tags = scenario.pickle.tags;
        StoryPanel.locatorAux.value = StoryPanel.storyOptionInDropdown.value.replace("{0}", ((!!tags.find(tag => { return tag.name === '@createBugStory'; }) ? "bug":"feature")));
        const optionSelectedInDropdown = await DriverFactory.myDriver.wait(until.elementLocated(StoryPanel.locatorAux));
        await optionSelectedInDropdown.click();
        const saveButton = await DriverFactory.myDriver.wait(until.elementLocated(StoryPanel.saveButton));
        await saveButton.click();
        await DriverFactory.myDriver.wait(until.elementIsVisible(addStoryButton), configuration.browser.timeout);
        let storyItem = await DriverFactory.myDriver.wait(until.elementLocated(StoriesTab.previewStoryItemRow));
        storyItem = await DriverFactory.myDriver.wait(until.elementTextContains(storyItem, this.firstStoryBugName));
        await storyItem.click();
    }
});`

Reply:
se realizo el cambio